### ISSUE TITLE
local vars don't show correct values when debugging

### ISSUE THREAD

**Issue by josesimoes** [microsoft/vscode-cpptools#903]
This is after the latest update to v0.12.1.
I have 32bits vars but the window with local vars is only showing 16bits values (despite the variable type is correctly showing using the mouse over).
I've reverted the extension to v0.12.0 and I can confirm that I can see the expected values debugging the same code, so it's not something related with the code I'm using.

**Comment by pieandcakes** [comment]
@josesimoes What OS and which debugger are you using?

**Comment by josesimoes** [comment]
I'm running Windows 10 (latest build), OpenOCD 0.10.0 and GNU gdb (GNU Tools for ARM Embedded Processors 6-2017-q1-update)

**Comment by pieandcakes** [comment]
Can you post a screenshot or a code example on what I need to do to see what you are seeing? I didn't make any changes around variables being displayed so I'm at a loss as to why they are showing 16 vs 32bit values.

**Comment by josesimoes** [comment]
I'll have to reinstall the latest version then...
I did notice that on the new version I can see values in hexadecimal format so something must be different from the latest version. (although this was in the main window, with mouse over, not in the variables window...)

**Comment by josesimoes** [comment]
@pieandcakes I was installing the new version again to take those screenshots for you when I noticed the update for VS Code that was published today. I updated it.
In the process the extension was updated too.
And I had to restart Windows because of an update too.
In the end I have the latest VS Code version _and_ the latest CPP tools extension version and everything seems to be working fine. I can see the variables content just fine.

Something must have happened in the update process that caused that.
Thanks for looking into this and keep up the ***AWESOME*** work on this extension! 😄
